mantidimaging/core/reconstruct/cil_recon.py

```diff
@@ -412,6 +412,6 @@ def full(images: ImageStack,
 
 def allowed_recon_kwargs() -> dict[str, list[str]]:
     return {
-        'CIL: PDHG-TV':
+        'CIL_PDHG-TV':
```

Comment:
'CIL_PDHG_TV' to be consistent

Reply:
I chose to use a hyphen between PDHG and TV as apposed to an underscore to show that TV is a configuration of the PDHG algorithm and directly related to the algorithm, and CIL is the library, not the algorithm. I am very happy to update if you feel that I should use an underscore for both 🙂

Reply:
That makes more sense.